src/Microsoft.Extensions.FileProviders.Abstractions/NullFileProvider/NullFileInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Actually I think we can also use `NoopChangeToken` and `NotFoundDirectoryContents` as well.